### ISSUE TITLE
Missing Multi-User functionality

### ISSUE THREAD

**Issue by rosscdh** [giantswarm/prometheus-pingdom-exporter#9]
Hi there,

firstly fantastic app thanks for the implementation. I was wondering if there were plans to support the https://www.pingdom.com/resources/api#multi-user+authentication which requires another header to be passed in.

It seems to already be supported in go-pingdom: https://github.com/russellcardullo/go-pingdom/blob/57ad1558c8041fb5bf0fea4bbd01f01a56770400/pingdom/pingdom.go#L40

Thanks for your efforts.

**Comment by rosscdh** [comment]
Seems to require the use of 

```
	client := pingdom.NewMultiUserClient(
		flag.Arg(1),
		flag.Arg(2),
		flag.Arg(3),
		flag.Arg(4),
	)
```

perhaps when passing in an optional flag?

```
	serverCmd.Flags().String(&accountEmail, "accountEmail", '', "The accountEmail to user for multi-user accounts")
```

Sorry am not a go programmer otherwise id take a stab at this functionality

**Comment by freeseacher** [comment]
looks like fixed with #14

**Comment by oponder** [comment]
Assuming this issue is resolved now, feel free to reopen or open another one.
